.gitattributes

```
@@ -1,2 +1,4 @@
-*.mat linguist-vendored
 *.ipynb linguist-vendored
+*.m linguist-vendored
+*.h5 linguist-vendored
+*.out linguist-vendored
```

```suggestion
*.out linguist-vendored
*.mat linguist-vendored
```
Im pretty sure there are .mat data files, and I don’t think it hurts. But you’re correct to add ‘.m’ files.